x/gov/keeper/vote.go

```diff
@@ -53,6 +54,15 @@ func (keeper Keeper) GetVotes(ctx sdk.Context, proposalID uint64) (votes types.V
 	return
 }
 
+func (keeper Keeper) GetVotesPaginated(ctx sdk.Context, proposalID uint64, page, limit int) (votes types.Votes) {
```

Comment:
I would remove this method and move the pagination logic within the querier handler like other modules do.

Reply:
i checked proposals method when i was  adding this one - https://github.com/cosmos/cosmos-sdk/blob/master/x/gov/keeper/proposal.go#L114-L148

Reply:
I'm not sure I follow? This method is not needed. Pagination can simple be done in `queryVotes`.

Reply:
i mean that when i was adding this method i intentionally made it consistent with that one (e.g. kept pagination in keeper)

Reply:
I see. Yeah, but this isn't necessary in this case.